flink-connectors/flink-connector-hive/src/main/java/org/apache/flink/table/endpoint/hive/HiveServer2Endpoint.java

```diff
@@ -869,10 +868,6 @@ private void buildTThreadPoolServer() {
                                     .inputProtocolFactory(
                                             new TBinaryProtocol.Factory(
                                                     true, true, maxMessageSize, maxMessageSize))
-                                    .requestTimeout(requestTimeoutMs)
```

Comment:
could you add clarification in the PR description for the reason of removing those code?

Reply:
methods and args were removed in new thrift versions. you can check HIVE-25098 for details.

Reply:
please leave review comments inline, I will summarize all conversions into the PR description eventually.

Reply:
Thanks for the clarification.